code/modules/mob/living/simple_animal/bot/ed209bot.dm

```diff
@@ -101,30 +103,32 @@
 	dat += hack(user)
 	dat += showpai(user)
 	dat += text({"
-<TT><B>Security Unit v2.6 controls</B></TT><BR><BR>
-Status: []<BR>
-Behaviour controls are [locked ? "locked" : "unlocked"]<BR>
-Maintenance panel panel is [open ? "opened" : "closed"]<BR>"},
+	<TT><B>Security Unit v2.6 controls</B></TT><BR><BR>
+	Status: []<BR>
+	Behaviour controls are [locked ? "locked" : "unlocked"]<BR>
+	Maintenance panel panel is [open ? "opened" : "closed"]<BR>"},
 
-"<A href='?src=[UID()];power=1'>[on ? "On" : "Off"]</A>" )
+	"<A href='?src=[UID()];power=1'>[on ? "On" : "Off"]</A>" )
 
 	if(!locked || issilicon(user) || user.can_admin_interact())
+		dat += text({"Auto Patrol[]<BR>"},
```

Comment:
Please don't add more text( procs. Change this to `dat += "Auto Patrol <A href='?src=[UID()];operation=patrol'>[auto_patrol ? "On" : "Off"]</A><BR>"`